modules/gathering-log-data.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * updating/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-log-data_{context}"]
+= Gathering log data
+
+To gather data from your cluster, including log data, you must learn how to leverage the `oc adm must-gather` command. You can find information and instructions on how to gather data from your cluster in the link:https://docs.openshift.com/container-platform/4.13/support/gathering-cluster-data.html[Gathering data about your cluster] section.
```

Comment:
Is there a way to link here without pinning 4.13?

Reply:
And micro-nit for "you must learn", learning about `must-gather` is a prereq, but to deliver this section's advertised `Gather log data`, you need to actually run `oc adm must-gather` ;)

Reply:
can totally adjust the wording :D

Reply:
The "ideal" way to link to that page (given the limitations of our docs – it's actually pretty unfortunate that we need to do it this way) would be to just reference the name of the page in this module, I believe putting the name in italics, and then to put an additional resources section right under where this module is included in the assembly. Then the xref to that page can be linked in the additional resources section.

[Here's an example](https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.html#:~:text=See%20Image%20set%20configuration%20parameters,About%20the%20OpenShift%20Update%20Service) of what I'm talking about. If you look at the [assembly file](https://github.com/openshift/openshift-docs/blob/main/updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc?plain=1#L106C1-L114), the module included on line 107 is where the pages are referenced in italics, and then lines 109-114 are where the xrefs are actually linked.

Reply:
Appreciate the feedback! Fixed it :)